netbox/extras/views.py

```diff
@@ -1189,3 +1190,73 @@ def post(self, request):
         rendered = render_markdown(form.cleaned_data['text'])
 
         return HttpResponse(rendered)
+
+
+#
+# Config Revision
+#
+
+
+class ConfigRevisionListView(generic.ObjectListView):
+    queryset = ConfigRevision.objects.all()
```

Comment:
ConfigRevisions are listed with the oldest first; it would make more sense to list the newest (current) revision first. Ideally this should be set by `Meta.ordering` on the model.